test/test_model_exports_to_core_aten.py

```diff
@@ -1,6 +1,8 @@
 # Owner(s): ["oncall: mobile"]
 import copy
 
+import pytest
```

Comment:
Is this file relevant to the PR? It doesn't seem like it's one of the executorch tests run

Reply:
The test is supposed to run on Linux in general to provide some cover for Executorch before this change, but it turns out that it wasn't run at all and was broken without anyone knowing.  I could move it to another PR if you think it's unrelated (I agree that it's indeed kind of unrelated)